specs/ScreenCaptureRequested.md

```diff
@@ -0,0 +1,357 @@
+                            [this](ICoreWebView3Frame* sender,
+                               ICoreWebView2ScreenCaptureRequestedEventArgs* args) -> HRESULT
+                            {
+
+                                    // Get Frame Info
+                                    wil::com_ptr<ICoreWebView2FrameInfo> frameInfo;
+                                    CHECK_FAILURE(args->get_FrameInfo(&frameInfo));
+
+                                    // Frame Source
+                                    wil::unique_cotaskmem_string frameSource;
+                                    CHECK_FAILURE(frameInfo->get_Source(&frameSource));
+
+                                    // If the host app wants to cancel the request for a specific source
+                                    if (frameSource = "https://developer.microsoft.com/en-us/microsoft-edge/webview2/") 
+                                    {
+                                        CHECK_FAILURE(args->put_Cancel(TRUE));
+                                    }
+
+                                    // Let CoreWebView2 handler know the event is already handled
+
+                                    // In the case of an iframe requesting permission to use Screen Capture, the default 
+                                    // behavior is to first raise the ScreenCaptureRequested event off of the 
+                                    // CoreWebView2Frame and invoke it's handlers, and then raise the event off the 
+                                    // CoreWebView2 and invoke it's handlers. However, If we set Handled to true on the
+                                    // CoreWebView2Frame event handler, then we will not raise the
+                                    // ScreenCaptureRequested event off the CoreWebView2.
+
+                                    CHECK_FAILURE(args->put_Handled(true));
+                                    return S_OK;
+                            })
+                            .Get(),
+                        &m_screenCaptureRequestedToken));
+                }
+                return S_OK;
+        }).Get(),
+        &m_FrameCreatedToken));
+}
+```
+
+## C#: Registering Screen Capture Requested Handler
+```c#
+private WebView2 webView;
+webView.CoreWebView2.ScreenCaptureRequested += (sender, screenCaptureArgs) =>
+{   
+    // Get Frame Info
+    wil::com_ptr<ICoreWebView2FrameInfo> frameInfo;
+    frameInfo = screenCaptureArgs.frameInfo
+
+    // Frame Source
+    string frameSource;
+    frameSource = frameInfo.Source;
+
+    // If the host app wants to cancel the request from a specific frame
+    if (frameSource == "https://developer.microsoft.com/en-us/microsoft-edge/webview2/")
+    {
+        screenCaptureArgs.Cancel = true;
+    }
+}
+```
+
+
+## C#: Registering IFrame Screen Capture Requested Handler
+```c#
+private WebView2 webView;
+
+m_webview.CoreWebView2.FrameCreated += (sender, frameCreatedArgs) =>

```

Comment:
Inconsistent variable name 'webView'/'m_webView'.

Reply:
similar issue to above with member / local naming and logic. Please fix similar to above.